Codigo/AI_NPC.bas

```diff
@@ -379,6 +379,9 @@ Private Sub AI_AtacarUsuarioObjetivo(ByVal AtackerNpcIndex As Integer)
                 If NpcLanzaSpellInmovilizado(AtackerNpcIndex, .TargetUser.ArrayIndex) Then
                     Call ChangeNPCChar(AtackerNpcIndex, .Char.Body, .Char.Head, tHeading)
 112                 Call NpcLanzaUnSpell(AtackerNpcIndex)
+                    If EstaPegadoAlUsuario Then
```

Comment:
esto no puede hacer que un npc tire un hechizo y pegue al mismo tiempo?

Reply:
If IsFeatureEnabled("Magic_and_Punch") And EstaPegadoAlUsuario And (RandomNumber(1, 100) <= 25) Then
                        Call NpcAtacaUser(AtackerNpcIndex, MapData(ComputeNextHeadingPos(AtackerNpcIndex).Map, ComputeNextHeadingPos(AtackerNpcIndex).x, ComputeNextHeadingPos(AtackerNpcIndex).y).UserIndex, tHeading)
                    End If

Reply:
[TOGGLE7]
name=Magic_and_Punch
value=1

Reply:
si entra muchas veces menos o minguna.

Reply:
para que es ese random number?